chia/full_node/mempool.py

```diff
@@ -1,18 +1,26 @@
 from __future__ import annotations
 
+import sqlite3
+from dataclasses import dataclass
 from datetime import datetime
 from enum import Enum
-from typing import Dict, List, Optional
-
-from sortedcontainers import SortedDict
+from typing import Dict, Iterator, List, Optional
 
+from chia.consensus.cost_calculator import NPCResult
 from chia.full_node.fee_estimation import FeeMempoolInfo, MempoolInfo, MempoolItemInfo
 from chia.full_node.fee_estimator_interface import FeeEstimatorInterface
-from chia.types.blockchain_format.coin import Coin
 from chia.types.blockchain_format.sized_bytes import bytes32
 from chia.types.clvm_cost import CLVMCost
 from chia.types.mempool_item import MempoolItem
-from chia.util.ints import uint64
+from chia.types.spend_bundle import SpendBundle
+from chia.util.chunks import chunks
+from chia.util.db_wrapper import SQLITE_MAX_VARIABLE_NUMBER
+from chia.util.ints import uint32, uint64
+
+# We impose a limit on the fee a single transaction can pay in order to have the
+# sum of all fees in the mempool be less than 2^63. That's the limit of sqlite's
+# integers, which we rely on for computing fee per cost as well as the fee sum
+MEMPOOL_ITEM_FEE_LIMIT = 2**50
```

Comment:
Would be nice to compute this from `(2^63)` and `MAX_BLOCK_COST_CLVM * BLOCK_SIZE_LIMIT_FACTOR`.

But this comment is more than sufficient.